text/41_advanced_unit_testing.md

```diff
@@ -451,9 +452,10 @@ public class ReflectionFun {
         try {
             System.out.println("Call public method (printQuack):");
             Method method = ReflectionFun.class.getMethod("printQuack");
+            ReflectionFun rf = new ReflectionFun();
             Object returnValue = method.invoke(rf);
-        } catch (NoSuchMethodException nsmex) {
-            System.err.println("No such method!");
+        } catch (NoSuchMethodException|IllegalAccessException|InvocationTargetException ex) {
```

Comment:
I tried to compile this example in jdk 1.8 and it was mad that these exceptions weren't caught.